packages/trace-viewer/src/traceModel.ts

```diff
@@ -116,74 +119,70 @@ export class TraceModel {
     return this._snapshotStorage!;
   }
 
-  private _build() {
-    this.contextEntry!.actions.sort((a1, a2) => a1.startTime - a2.startTime);
-    this.contextEntry!.resources = this._snapshotStorage!.resources();
```

Comment:
We never set `resources` anymore, perhaps remove them from the type definition?

Reply:
We do populate it incrementally: contextEntry.resources.push(event.snapshot);